versions.tf

```diff
@@ -14,5 +14,5 @@
  * limitations under the License.
  */
 terraform {
-  required_version = ">=0.12.6"
+  required_version = ">=0.13"
```

Comment:
Since we are bumping `>=0.13` would be great to get module attribution  in as well. https://github.com/terraform-google-modules/terraform-google-project-factory/pull/513